src/serverrpc.cpp

```diff
@@ -233,7 +233,7 @@ QJsonValue CServerRpc::SerializeRegistrationStatus ( ESvrRegStatus eSvrRegStatus
         return "registered";
 
     case SRS_SERVER_LIST_FULL:
-        return "directory_server_full";
+        return "directory_server_full"; // TODO - rename to "server_list_full"
```

Comment:
Why not directory_(list)_full?

Reply:
Because it's called the "server list".

Reply:
Then the terminology should be added to the style and tone guide: https://jamulus.io/contribute/Style-and-Tone

Reply:
OK, see https://github.com/jamulussoftware/jamuluswebsite/pull/910.